unicon_backend/cli.py

```diff
@@ -5,9 +5,54 @@
 from rich.syntax import Syntax
 from rich.table import Table
 
+from unicon_backend.dependencies.project import role_permissions
+from unicon_backend.lib.permissions.permission import (
+    debug_list_tuples,
+    delete_all_permission_records,
+    init_schema,
+    permission_create,
+)
+
 rich_console = Console()
 app = typer.Typer(name="Unicon 🦄 CLI")
 
+permify_app = typer.Typer()
+app.add_typer(permify_app, name="permify")
+
+
+@permify_app.command(name="init")
+def init_permify():
+    """Sends the schema file to permify."""
+    schema_version = init_schema("unicon_backend/lib/permissions/unicon.perm")
```

Comment:
nit: use `importlib.resources` to get path of `.perm` file? this is so that we wont have to deal with path issues when we package `unicon_backend` as a package